libs/langchain-community/src/embeddings/bytedance_doubao.ts

```diff
@@ -0,0 +1,175 @@
+import { getEnvironmentVariable } from "@langchain/core/utils/env";
+import { Embeddings, type EmbeddingsParams } from "@langchain/core/embeddings";
+import { chunkArray } from "@langchain/core/utils/chunk_array";
+
+export interface ByteDanceDoubaoEmbeddingsParams extends EmbeddingsParams {
+  /** Model name to use */
+  modelName: string;
```

Comment:
We are standardizing as `model` instead of `modelName`

Reply:
done